README.md

```diff
@@ -1 +1,100 @@
-# my-portforlio-project
\ No newline at end of file
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 [Hello microverse]
+Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] Please make sure you used the correct [readme template](https://github.com/microverseinc/readme-template). It seems like this: 
![image](https://user-images.githubusercontent.com/91486791/235244294-bdf2a1e4-5db0-4096-b90b-86d8866a20ed.png)
should be like this: 
![image](https://user-images.githubusercontent.com/91486791/235244482-e9cab6f4-8628-4625-a8ec-3c92d9ea45f7.png)

Reply:
Nice job on trying to use the README template 📖 but the formatting is not quite right yet, I'd suggest replacing what you have with the [template](https://github.com/microverseinc/readme-template) and then editing the information, remember all the sections are required except for the `FAQ`, some sections and subsections can be removed ❌ as they are not relevant yet to this project such as `Live Demo`, `Usage`, `Run Tests` and `Deployments`. 🤓 

PS: This is an invalidation reason 🙅🏽 (the first reviewer should've invalidated) as it is mandatory to have a good README for a review to be performed, make sure that the next time you submit a PR, your README is professional. If you need help, feel free to tag me here or contact me on Slack. 💯